model_analyzer/triton/model/model_config.py

```diff
@@ -0,0 +1,166 @@
+# Copyright (c) 2021, NVIDIA CORPORATION. All rights reserved.
```

Comment:
This file can be removed.

Reply:
Will do. But I think #78 will need to be merged first right?

Reply:
Right, sorry I thought it is not used anywhere.